src/control/control.h

```diff
@@ -122,16 +122,13 @@ int dt_control_is_key_accelerators_on(struct dt_control_t *s);
 // All the accelerator keys for the key_pressed style shortcuts
 typedef struct dt_control_accels_t
 {
-  GtkAccelKey filmstrip_forward, filmstrip_back, lighttable_up, lighttable_down,
-    lighttable_right, lighttable_left, lighttable_pageup, lighttable_pagedown,
-    lighttable_start, lighttable_end, lighttable_sel_up, lighttable_sel_down,
-    lighttable_sel_right, lighttable_sel_left, lighttable_sel_pageup,
-    lighttable_sel_pagedown, lighttable_sel_start, lighttable_sel_end,
-    lighttable_center, lighttable_preview, lighttable_preview_display_focus,
-    lighttable_timeline, lighttable_preview_zoom_100, lighttable_preview_zoom_fit,
-    global_focus_peaking, global_sideborders, global_accels_window,
-    darkroom_preview, slideshow_start, darkroom_skip_mouse_events, global_collapsing_controls,
-    slideshow_view;
+  GtkAccelKey filmstrip_forward, filmstrip_back, lighttable_up, lighttable_down, lighttable_right, lighttable_left,
+      lighttable_pageup, lighttable_pagedown, lighttable_start, lighttable_end, lighttable_sel_up,
+      lighttable_sel_down, lighttable_sel_right, lighttable_sel_left, lighttable_sel_pageup,
+      lighttable_sel_pagedown, lighttable_sel_start, lighttable_sel_end, lighttable_center, lighttable_preview,
+      lighttable_preview_display_focus, lighttable_timeline, lighttable_preview_zoom_100,
+      lighttable_preview_zoom_fit, global_focus_peaking, global_sideborders, global_accels_window, global_guides,
```

Comment:
You don't need this because you're actually connecting a closure to the accelerator, right?

These copies of assigned accelerator keys are only needed when you manually compare a pressed key to an assigned shortcut (in a key_pressed method, for example). You don't seem to be doing that, unless I'm missing something?

There are quite a few other unnecessary/unused ones btw. A lot of cargo culting in this particular area.

Reply:
right ! that was just stupid copy-paste from focus peaking (I guess that's not useful there too, but I prefer to keep this PR for guides only)

Reply:
Don't worry about cleaning that up; I'm doing it for #8078 anyway and it doesn't really hurt anything. Except the brain of the next person trying to understand it ;-)